aces/Loris-MRI
# [Perl] [tools] Add new script that replicates RaisinBread dataset

### Description
Added a new script that makes a local copy of the RaisinBread data set. Instead of copying files from the original data set, symlinks are created instead.

### How to test

1. Create a folder where the copied data set will be stored. Ensure that the permissions of the folder are set so that script can create new directories and files within the folder. The recommended location is in `/data/`:
`sudo mkdir /data/data-raisinbread`
`sudo chown lorisadmin:www-data /data/data-raisinbread`
`chmod 750 /data/data-raisinbread`
2. Run the script (located in `tools/`). The script requires two arguments. The first is the location of the original dataset and the second is the location of the newly created directory. Below is an example using the directory created in step 1.
`tools/replicate_raisinbread_for_mcin_dev_vm.pl /data-raisinbread/ /data/data-raisinbread/`
